client/src/boot/BootRoutes.js

```diff
@@ -92,6 +94,22 @@ class BootRoutes {
       </ApolloProvider>,
       document.getElementsByClassName('cms-content')[0]
     );
+
+    history.listenBeforeUnload(() => {
+      if (this.shouldConfirmBeforeUnload()) {
+        return i18n._t('Admin.CONFIRMUNSAVEDSHORT');
```

Comment:
Can you please use the default string as well?

`return i18n._t('Admin.CONFIRMUNSAVEDSHORT', 'WARNING: Your changes have not been saved.');`